onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -4049,10 +4060,12 @@ def aten_repeat_interleave(
     raise NotImplementedError()
 
 
+@torch_op("aten::reshape")
 def aten_reshape(self: TensorType, shape: INT64) -> TensorType:
     # reshape(Tensor(a) self, SymInt[] shape) -> Tensor(a)
 
-    raise NotImplementedError()
+    shape_int64 = op.Cast(shape, to=7)  # Reshape only support INT64 as 'shape'
```

Comment:
Change all 7 -> INT64.dtype to avoid magic numbers

Reply:
fixed